### PR TITLE
Module 1: Portfolio Mobile Menu

### PR DESCRIPTION
**Pull request by ikennarichard**
## Module 1 (Week 4): Portfolio Mobile Menu
For this project, I
- Made use of the HTML, CSS and JavaScript Linters
- Used JavaScript to display/hide the mobile menu when the hamburger button is clicked
- Edited the 'Welcome' logo animation
- Worked with my coding partner to update page styles
- Made the page to go to the correct section of the mobile menu options
- I used JavaScript to toggle the hamburger button
- Added .gitattributes file to prevent LF to CRLF line ending conversions for .js files which caused eslint checks to fail.
I already merged my initial pull request for this project which contains my coding partner's commits, I was asked to make a new one by the previous reviewer after it failed the linters check, here is the link to the initial pull request https://github.com/ikennarichard/my-portfolio/pull/9

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @ikennarichard, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the student's success team and they'll permit you to work solo or look for a suitable partner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Comment by ikennarichard** [comment]
Please should my coding partner make commits on this new pull request @topeogunleye

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+*.js text eol=lf
```
Comment (bobb-Rob) [inline .gitattributes:1]: - [ ] [OPTIONAL] Well done for implementing this project requirement nicely. I have a little suggestion that can help improve this project further. When the user clicks the menu icon, the menu appears, but it scrolls along when the page is scrolled up. Ideally, this menu display should be fixed i.e not scrollable.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @ikennarichard, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**